ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/vulnerabilities/index.tsx

```diff
@@ -138,34 +146,47 @@ const renderSubComponent = ({
 }: {
   row: Row<VulnerabilityWithIdentifier>;
 }) => {
-  const references = row.original.vulnerability.references;
+  const vulnerability = row.original.vulnerability;
 
   return (
     <div className='flex flex-col gap-4'>
       <div className='font-semibold'>Description</div>
       <div className='whitespace-pre-line italic text-muted-foreground'>
-        {row.original.vulnerability.description}
+        {vulnerability.description || 'No description.'}
       </div>
-      <Card className='rounded-lg border px-2'>
-        <CardHeader>
-          <CardTitle>References</CardTitle>
-        </CardHeader>
-        <CardContent>
-          {references.map((reference, index) => (
-            <div key={index} className='flex gap-2'>
-              <div>{reference.severity}</div>
-              <div>{reference.scoringSystem}</div>
-              <Link
-                className='font-semibold text-blue-400 hover:underline'
-                to={reference.url}
-                target='_blank'
-              >
-                {reference.url}
-              </Link>
-            </div>
+      <div className='font-semibold'>References</div>
+      <Table>
```

Comment:
Is there a specific reason for not using DataTable component here?

Reply:
The parent component is already a DataTable, and I saw no reason for complicating by inserting another DataTable inside it, as in my view there's nothing it would bring to this view.

Reply:
In short, I don't see the added value, compared to the level of complexity it would introduce to this page.

Reply:
Oh, didn't notice it was the subrow. Absolutely makes sense.